content/docs/02.5-architecture/01-architecture-overview.md

```diff
@@ -19,6 +19,13 @@ Palette is available in three flexible deployment models:
 
 * **Self-hosted**: The management plane is hosted in the customer’s environment. It can be the customer’s on-prem VMware vSphere, OpenStack, bare metal, or in a public cloud using the customer’s cloud account.
 
+<br />
+
+## Product Security
```

Comment:
It makes sense to have a `Product Security` sub-section here, but the content of the section doesn't make a lot of sense in here: I'd suggest to have an introduction text about security, maybe something like:

```
At Spectro Cloud, we recognize the importance of robust security measures in today's rapidly evolving digital landscape. As the provider of our cutting-edge SaaS and self-hosted product, Palette, our commitment to safeguarding our customers' data and ensuring the integrity of our services is paramount. Learn more about what we do in terms of security in our Security section.
```

By the way, the link here is broken, it's a 404. ;)

Reply:
@cloudmaniac I must've fixed this before, along with the link but didn't mark this resolved. The text is as you describe and the link is fixed.